spec/API_specification/array_api/linalg.py

```diff
@@ -3,25 +3,40 @@
 
 def cholesky(x: array, /, *, upper: bool = False) -> array:
     """
-    Returns the lower (upper) Cholesky decomposition x = LLᵀ (x = UᵀU) of a symmetric positive-definite matrix (or a stack of matrices) ``x``, where ``L`` is a lower-triangular matrix or a stack of matrices (``U`` is an upper-triangular matrix or a stack of matrices).
+    Returns the lower (upper) Cholesky decomposition of a complex Hermitian or real symmetric positive-definite matrix ``x``.
 
-    ..
-      NOTE: once complex numbers are supported, each square matrix must be Hermitian.
+    If ``x`` is real-valued, let :math:`\\mathbb{K}` be the set of real numbers $\\mathbb{R}$, and, if ``x`` is complex-valued, let $\\mathbb{K}$ be the set of complex numbers $\\mathbb{C}$.
+
+    The lower **Cholesky decomposition** of a complex Hermitian or real symmetric positive-definite matrix :math:`x\\ \\in\\ \\mathbb{K}^{n \\times n}` is defined as
+
+    .. math::
+       x = LL^{H} \\qquad \\text{L $\\in\\ \\mathbb{K}^{n \\times n}$}
+
+    where :math:`L` is a lower triangular matrix and :math:`L^{H}` is the conjugate transpose when :math:`L` is complex-valued and the transpose when :math:`L` is real-valued.
+
+    The upper Cholesky decomposition is defined similarly
+
+    .. math::
+       x = UU^{H} \\qquad \\text{U $\\in\\ \\mathbb{K}^{n \\times n}$}
```

Comment:
This is better done by starting the docstring with `r"""`, that avoids having to use the awful double backslashes.

Reply:
Changed.